### PR TITLE
Tests/hdrp/shadergraph hdrp automated

### PR DESCRIPTION
**Pull request by zadi0648**
### Purpose of this PR

- Added Shader Graph tests to Build Settings so that they are actually used :P
- Made some of the test less sensitive to failure (as they failed without good cause)
- Fixed Anisotropic Tangent test
- Added Distortion test to Transparent 

---
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.

---
### Testing status

- Green on my machine

**Katana Tests**:  Hopefully these specific test pass by the end of the weekend and before Europe's Monday: https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=tests%2Fhdrp%2Fshadergraph-hdrp-automated&unity_branch=2018.3%2Fstaging&automation-tools_branch=master

---
### Overall Product Risks
**Technical Risk**: None

**Halo Effect**: None

### CODE REVIEW

**Comment by RemyUnity** [comment]
Apparently the tests failed on Katana @zadi0648

File: TestProjects/HDRP_Tests/ProjectSettings/EditorBuildSettings.asset [hunk 1]
```diff
@@ -173,8 +173,21 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/GraphicTests/Scenes/2x_Lighting/2306_Contact_Shadow_SpotLight.unity
     guid: a8ecf1e3cee864a48bd6bca2c797fdb3
-    path: Assets/GraphicTests/Scenes/9x_Other/9301_MotionVectorsOff-Forward-Dynamic.unity
```
Comment (sebastienlagarde) [inline TestProjects/HDRP_Tests/ProjectSettings/EditorBuildSettings.asset]: Hey, seems that you have removed this test by mistake?

Comment (zadi0648) [reply TestProjects/HDRP_Tests/ProjectSettings/EditorBuildSettings.asset]: Not sure what happened there. Should be added now?

**Comment by sebastienlagarde** [comment]
Now that we have create branch, trying to get katana green

**Comment by sebastienlagarde** [comment]
So, this test have passed graphic test on windows - katana on 2018.3. Which is good. Now that this PR have merged master into it, it is not possible anymore to launch katana on 2018.3, it will fail (master include screenshots only working on 2019.1). And inversely, the screenshots of this PR are for 2018.3, so they will fail on katana 2019.1.

I will be good with merging the PR (as currently katana 2019.1 is broken anyway), do the backport on 2018.3, then regenerate all the screenshots for master), I will dot it.

However this PR have fail Metal graphic test. And I am not sure if this come from this PR or not? I relaunch katana on the commit before the merge master to be sure, but if it fail, it will delay the PR as it will require investigation on your side.

**Comment by sebastienlagarde** [comment]
Katana green, merging.

**Pull request merged**